### DISCUSSION TITLE
无法使用groq

### DISCUSSION THREAD

**Discussion by LDR24** [nextai-translator/nextai-translator#1360] [Q&A]
<img width="443" alt="屏幕截图 2024-03-07 143454" src="https://github.com/openai-translator/openai-translator/assets/139335368/02b5b0a2-1467-4fec-bc97-289c63690f41">

**Comment by makaoxue** [comment]
是的，我也遇到这种情况

**Comment by LDR24** [comment]
😂，时好时不好的

makaoxue ***@***.***> 于 2024年3月10日周日 下午5:26写道：

**Comment by yetone** [comment]
应该是翻墙代理软件导致的或者是 groq 服务本身跪了
